static/app/components/draggableTabs/index.tsx

```diff
@@ -0,0 +1,38 @@
+import 'intersection-observer'; // polyfill
+
+import {useState} from 'react';
+import type {Key} from '@react-types/shared';
+
+import {DraggableTabList} from 'sentry/components/draggableTabs/draggableTabList';
+import {TabPanels, Tabs} from 'sentry/components/tabs';
+
+export interface Tab {
+  content: React.ReactNode;
+  key: Key;
+  label: string;
+  queryCount?: number;
+}
+
+export interface DragAndDropTabBarProps {
+  tabs: Tab[];
+  tempTabContent: React.ReactNode;
+}
+
+export function DraggableTabBar(props: DragAndDropTabBarProps) {
```

Comment:
Is this used for anything but the `.stories` file? If so I'd put it there. I'd expect this kind of implementation to be in the issues folder eventually since it is implementing DraggableTags in a specific way.

Reply:
Currently, no. It is only being used in the `.stories` file. 

Are you saying put it (this component) there (the issues folder)?  (if so, I agree, but just wanted to clarify since I'm not sure how to resolve the pro nouns haha)

Reply:
If this is meant to be used on the issues page (which I assume it is), then yes this should be in the `views/issueList` folder. Or if it's only meant for demo-ing on the storybook, put it in the `.stories` file